dash/App/Accounts/Add/AddAddress/index.js

```diff
@@ -52,9 +49,16 @@ class AddAddress extends React.Component {
     this.focusActive()
   }
 
-  create () {
-    this.setState({ index: ++this.state.index })
-    link.rpc('createFromAddress', this.state.address, 'Watch Account', (err) => {
+  setResolving() {
+    this.setState(({resolvingEns, ...state}) => ({
+      ...state,
+      resolvingEns: true
+    }))
```

Comment:
you don't need to add the previous state here, just update the properties you want to change and React will handle it
```suggestion
    this.setState({ resolvingEns: true })
```